build/docker/scripts/ubsan.sh

```diff
@@ -15,7 +15,7 @@ export CXX=clang++-3.8
 # undefined casting, aka "vptr".
 #
 # TODO: fix undefined vptr behavior and turn this option back on.
-export CFLAGS="-fsanitize=undefined -fno-sanitize-recover=undefined -fno-sanitize=vptr"
```

Comment:
We'll still want to include `-fno-sanitize-recover=undefined`, I think. When that option is set, a binary will crash on undefined behavior. When it is not set, the binary will print an error and continue.

Having it set is preferable, I think, so that undefined behavior causes the tests to fail.